dom/traversal/TreeWalker.html

```diff
@@ -0,0 +1,304 @@
+<!doctype html>
+<title>TreeWalker tests</title>
+<link rel="author" title="Aryeh Gregor" href=ayg@aryeh.name>
+<meta name=timeout content=long>
+<div id=log></div>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src=../common.js></script>
+<script>
+"use strict";
+
+// TODO .previousNode, .nextNode
+
+test(function() {
+    var walker = document.createTreeWalker(document);
+    assert_equals(walker.whatToShow, 0xFFFFFFFF, "whatToShow");
+    assert_equals(walker.filter, null, "filter");
+}, "createTreeWalker() parameter defaults");
```

Comment:
This test seems to be redundant with https://github.com/w3c/web-platform-tests/blob/master/dom/traversal/TreeWalker-basic.html#L90